client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -322,6 +322,9 @@ public class RssSparkConfig {
 
   public static final ConfigEntry<String> RSS_ACCESS_ID =
       createStringBuilder(new ConfigBuilder("spark.rss.access.id")).createWithDefault("");
+  public static final ConfigEntry<String> RSS_ACCESS_ID_PROVIDER_KEY =
```

Comment:
Could you use camel style?

Reply:
Done.

I renamed to `spark.rss.access.id.providerKey`.